english/bob-barker/1970s/1970s.md

```diff
@@ -2,20 +2,20 @@ You open your eyes and see yourself reflected back in a mirror.
 
 You have on a bright blue shirt whose collar is about twice the size it should be.  
 
-You look down and see that your jeans are bell bottoms fastended with an oversized buckle.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.